src/GeoPing.Api/Views/Account/Login.cshtml

```diff
@@ -18,23 +18,15 @@
                 <hr />
                 <div asp-validation-summary="All" class="text-danger"></div>
                 <div class="form-group">
-                    <label asp-for="Email"></label>
```

Comment:
We don't use razor views in the project, delete them as well as corresponding controller actions

Reply:
done